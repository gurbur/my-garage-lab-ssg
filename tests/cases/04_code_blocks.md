먼저 언어 타입이 지정된 C 코드 블록입니다.

```c
#include <stdio.h>

// main function
int main() {
    printf("Hello, world! *이것은 볼드가 아니다*");
    return 0;
}
```

다음은 언어 타입이 없는 일반 코드 블록입니다.

```
plain text
    - with indentation
# this is not a heading
```
