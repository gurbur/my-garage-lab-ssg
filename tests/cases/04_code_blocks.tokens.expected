[TEXT]{"먼저 언어 타입이 지정된 C 코드 블록입니다"}
[DOT]
[NEWLINE]
[NEWLINE]
[BACKTICK]
[BACKTICK]
[BACKTICK]
[TEXT]{"c"}
[NEWLINE]
[TEXT]{"#include <stdio.h>

// main function
int main() {
    printf("Hello, world! *이것은 볼드가 아니다*");
    return 0;
}
"}
[BACKTICK]
[BACKTICK]
[BACKTICK]
[NEWLINE]
[NEWLINE]
[TEXT]{"다음은 언어 타입이 없는 일반 코드 블록입니다"}
[DOT]
[NEWLINE]
[NEWLINE]
[BACKTICK]
[BACKTICK]
[BACKTICK]
[NEWLINE]
[TEXT]{"plain text
    - with indentation
# this is not a heading
"}
[BACKTICK]
[BACKTICK]
[BACKTICK]
[NEWLINE]
[EOF]
